include/AutomatableModelView.h

```diff
@@ -28,9 +28,14 @@
 
 #include "ModelView.h"
 #include "AutomatableModel.h"
+#include "AutomationTrack.h"
+#include "AutomationClip.h"
+#include "SongEditor.h"
+#include "Song.h"
```

Comment:
I should be possible to include forward declarations for most of these includes.

You have also included `Song` but forward declared it at the same time.

Reply:
I can't forward declare `AutomationTrack` and `AutomationClip` for some reason.

The compiler finds errors in `/build/src/lmmsobjs_autogen/ ... /include/AutomationEditor.h`

Reply:
Make sure they are forward declared within the correct namespace

Reply:
Thanks, that worked!